examples/mistral/config.yml

```diff
@@ -0,0 +1,69 @@
+base_model: mistralai/Mistral-7B-v0.1
+base_model_config: mistralai/Mistral-7B-v0.1
+model_type: MistralForCausalLM
```

Comment:
Should be able to use AutoModelForCausalLM

Reply:
Yes, but I think it would be better to specify for later parsing (aka is_mistral_derived..)